rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,763 @@
+  - url: http://127.0.0.1:1080
+    description: URL Used for Mock-Server Unit Tests
+# All routes are currently configured using an Authorization header.
+security:
+- BearerAuth: []
+paths:
+  /v1/config:
+    get:
+      tags:
+      - Configuration API
+      summary: List all catalog configuration settings
+      operationId: getConfig
+      description:
+        All REST catalogs will be initialized by calling this route. This route
+        will return at least the minimum necessary metadata to initialize the
+        catalog. Optionally, it can also include server-side specific overrides.
+        For example, it might also include information used to initialize this catalog
+        such as the details of the Http connection pooling, etc. This route might
+        also advertise information about operations that are not implemented
+        so that the catalog can eagerly throw or go about another way of performing
+        the desired action.
+      responses:
+        default:
+          description: Server-Specific Configuration Overrides
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/IcebergConfiguration'
+        "400":
+          description: Unknown Error
+        "401":
+          description: Invalid credentials provided
+  # This might be optional for now as it's not really supported in
+  # the normal Catalog spec, but we might want to include it for
+  # convenience.
+  /v1/catalogs/{catalog}:
+    parameters:
+      - name: catalog
+        in: path
+        required: true
+        description: Name of the catalog being configured
+        schema:
+          type: string
+          minLength: 1
+    post:
+      tags:
+         - Configuration API
+      summary: Persist catalog specific configuration, which can be retrieved
+        for later use.
+      operationId: postConfig
+      description:
+        Persist some catalog specific configurations, which will be returned by \
+        calls to /v1/config in the future. This is basically all of the data \
+        that would go into the Catalog's `initialize` call.
+      # TODO - Make this into a CatalogConfiguration
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Catalog'
+        required: true
+      responses:
+        '200':
+          description: OK
+        '401':
+          description: Unauthorized / Invalid Credentials Provided
+
+  # Could also consider /v1/tables/{identifier}
+  /v1/namespaces/{namespace}/tables/{table}:
+    parameters:
+      # Consider moving this to query parameters,so it can be more easily URL encoded
+      # in case of dots or special characters
+      - name: namespace
+        in: path
+        description: Namespace the table is in
+        required: true
+        schema:
+          type: string
+        examples:
+          singlepart_namespace:
+            value: "prod"
```

Comment:
I think using `prod` as the example will make people confuse this with catalogs. I would use something like `accounting` here and `accounting.tax` for the multipart.

Reply:
Removed `prod` from all of the example namespaces.